CuratedContent/EconomyOfOpenSourceSW.md

```diff
@@ -0,0 +1,28 @@
+# A look at the economic forces in Open Source Software
+
+The Slate Article *The Internet Was Built on the Free Labor of Open Source Developers. Is That Sustainable?* looks at the economics driving the Open Source movement and how that is putting those utilizing that software without investing in it at risk.
+
```

Comment:
Is this table a boilerplate section for any curated content article. I find it distracting to be embedded in the *middle* of the article. Any reason we shouldn't move it to end?

Reply:
IOt is from the template - I don't care for it's location either. Do we know who built the template? I would move this just after the "Contributed By".

Reply:
I am fine with whatever. Just wanted to raise the issue.